PreSignUpLambdaHook/index.mjs

```diff
@@ -1,35 +1,26 @@
-﻿import pg from "pg";
-import fs from "fs";
+﻿import pg from 'pg';
+import fs from 'fs/promises';
```

Comment:
Personal style, but I prefer double quotes for all strings
```suggestion
﻿import pg from "pg";
import fs from "fs/promises";
```

Reply:
Interesting... I've been using single-quotes for so long the double quotes look weird to me! I've generally based things on the airbnb style guide (https://github.com/airbnb/javascript) which chooses single quotes, and most JS I've seen in recent years matches.

Obviously there's very little in it... I think the main reason is that double-quotes turn up more frequently in stuff like HTML, so it slightly reduces the number of escapes you may need. In this very example, we need to use double quotes in the SQL query, and so single quotes avoid needing escapes!

Our JS script for compiling the CSS/SASS uses single quotes, so I think it's consistent in the codebase too.

Reply:
I will admit that it makes switching between JS and C# a bit more annoying, because you have to switch quote styles 😬